docs/cloud-soar/cloud-soar-delivery-2.md

```diff
@@ -0,0 +1,143 @@
+---
+id: cloud-soar-delivery-2
+title: Cloud SOAR Delivery 2
+sidebar_label: Cloud SOAR Delivery 2
+description: Learn about the features in the Cloud SOAR Delivery 2 release.   
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+This topic describes the features in the Cloud SOAR Delivery 2. 
+
+Delivery 2 of Cloud SOAR offers the following enhancements over our initial Cloud SOAR product:
+
+* New and improved features
+   * [Dashboards](#create-a-dashboard)
+   * [Reports](#create-a-report)
+
+ * Open Integration Framework updates
+      * [Integration Builder](#integration-builder) allows you to build integrations without needing to provide code
+     * Certified integrations allow you to customize JSON and table output schema
+     * Actions configuration during  playbook design is rearranged for easier use
+     * Integrations, and related action execution, can be done [in the cloud or through the Bridge](#cloud-or-bridge-execution). Only certified integrations can be executed in the cloud.
+* Architectural improvements
+     * Fully-functional in the Cloud (the Bridge is only required for custom integrations)
+     * User and profile management is in Sumo Logic core platform instead of Cloud SOAR
+     * Automatic scalability based on server load
```

Comment:
```suggestion
* New and improved features:
   * [Dashboards](#create-a-dashboard)
   * [Reports](#create-a-report)
 * Open Integration Framework updates:
     * [Integration Builder](#integration-builder) allows you to build integrations without needing to provide code
     * Certified integrations allow you to customize JSON and table output schema
     * Actions configuration during  playbook design is rearranged for easier use
     * Integrations and related action executions can be done [in the cloud or through the Bridge](#cloud-or-bridge-execution). Only certified integrations can be executed in the cloud.
* Architectural improvements:
     * Fully-functional in the Cloud (the Bridge is only required for custom integrations)
     * User and profile management is in Sumo Logic core platform instead of Cloud SOAR
     * Automatic scalability based on server load
```